### DISCUSSION TITLE
ignore versions gradle

### DISCUSSION THREAD

**Discussion by Rbsn-joses** [renovatebot/renovate#9339] [Request Help]
I'm trying to set up renovate (self-hosted) to ignore versions that have the suffix -dev, .dev, -rc, but I haven't been able to ignore it yet, it has brought versions dev, (1.1.0 to 1.4.1-dev )
I would like to know if how to configure renovate.json for it to ignore these versions

Could you write an example for me to follow?

**Comment by viceice** [comment]
Try [allowedVersions](https://docs.renovatebot.com/configuration-options/#allowedversions)

**Reply by Rbsn-joses** [reply]
I tried to use it, but I must be setting it wrong

{
  "ignoreUnstable": true ,
  "baseBranches": ["alpha"],
  "packageRules": [
    {
      "matchDatasources": ["gradle-version"],
      "matchPackagePatterns": ["^dep"],
      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[0-9]$/",
      "versionScheme": "regex:^(?<compatibility>.*)-v?(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)?$"
    }
  ],
  "gradle-lite": {
    "enabled": true
  }
  
  
}

**Reply by rarkins** [reply]
Why did you add `matchPackagePatterns` as well as `versionScheme`? And is this for a gradle dependency, or for gradle itself?

**Reply by Rbsn-joses** [reply]
I was testing all the ways, I have some doubts about the doc in some points, so I am implementing some settings that may not work at all. it would be for private dependencies in the file, build.gradle, it brings some dev versions so I'm testing to see if the renovate bar these versions get the latest stable version

**Reply by Rbsn-joses** [reply]
I want to understand how to make this configuration correctly

**Reply by rarkins** [reply]
If it's for dependencies in build.gradle then the datasource to match is `maven`, not `gradle-version`. I would not complicate things with `versionScheme` and remove it instead.

**Reply by Rbsn-joses** [reply]
It's works, thank you, the renovate is amazing
